conf/locale/locale_en-US.ini

```diff
@@ -186,6 +186,13 @@ TeamName = Team name
 AuthName = Authorization name
 AdminEmail = Admin email
 
+NewBranchName = New branch name
```

Comment:
I have no idea why these are camel case and not lower case with underscores....I usually always do the latter.